start

```diff
@@ -1,10 +1,23 @@
 #!/bin/sh
 set -e
 
+echo "Starting..."
+
+# Fix permissions on SSH private key.
+chmod 600 /app/data/ssh/id_rsa
```

Comment:
This file does not exist on first start, so this fails:

```
chmod: cannot access '/app/data/ssh/id_rsa': No such file or directory
```

See https://travis-ci.org/WeblateOrg/docker/builds/558894136#L2502